docs/_includes/content.md

```diff
@@ -3358,10 +3357,10 @@ unstable_features = ["CTRL_C_HANDLING"]
 
 Below is a list of currently existing unstable features
 
-* CTRL_C_HANDLING - Adds ctrl-c handler which will stop any currently running command invoked by the current task and exit cargo-make
+* **CTRL_C_HANDLING** - Adds <kbd>ctrl-c</kbd> handler, which will stop any currently running command invoked by the current task and exit cargo-make
 
 <a name="usage-cli"></a>
-### Cli Options
+### CLI Options
```

Comment:
can you also update the nav.md for this?

Reply:
Sure!